PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -25,6 +25,8 @@
         'keep recoCaloClusters_reducedEgamma_*_*',
         'keep EcalRecHitsSorted_reducedEgamma_*_*',
 
+        'keep *_egmPhotonPUPPIIsolationForPhotons_*_*', 
```

Comment:
the reason to add that was that this was the only way I could bring this running in the unscheduled mode. I think it would be ok not to keep this value map in miniAOD. I'm not sure how this works in the unscheduled mode. Please clarify